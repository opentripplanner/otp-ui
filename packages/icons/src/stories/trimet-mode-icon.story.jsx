import React from "react";

import TrimetModeIcon from "../trimet-mode-icon";

import ModeIconRenderer from "./mode-icon-renderer";

export default {
  title: "Icons/TrimetModeIcon",
  component: TrimetModeIcon
};

export const TrimetModeIconExamples = () => (
  <ModeIconRenderer component={TrimetModeIcon} />
);
