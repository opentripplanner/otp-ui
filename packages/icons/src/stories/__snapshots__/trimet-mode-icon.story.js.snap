// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Icons/TrimetModeIcon TrimetModeIconExamples smoke-test 1`] = `
<table class="icon-renderer__StyledTable-sc-epf3ui-0 fqtAeK">
  <thead>
    <tr>
      <th>
        Mode Type
      </th>
      <th>
        Result
      </th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>
        BICYCLE
      </td>
      <td>
        <div>
          <svg viewbox="0 0 390 390">
            <path d="M117.1 116.3h33.8c4.7 0 8.4-3.8 8.4-8.4 0-4.7-3.8-8.4-8.4-8.4h-33.8c-4.7 0-8.4 3.8-8.4 8.4-.1 4.6 3.7 8.4 8.4 8.4z">
            </path>
            <path d="M316.4 162.3c-10 0-19.5 2-28.3 5.6-2.5-6.3-4.6-11.9-6-15.8 0 0 0-.1-.1-.2l-24.7-65.8c-1.2-3.3-4.4-5.5-7.9-5.5H217c-4.7 0-8.4 3.8-8.4 8.4 0 4.7 3.8 8.4 8.4 8.4h26.7l10.4 27.9h-113c-3 0-5.6 1.6-7.1 3.9l-25.8 40.6c-9.9-4.8-20.9-7.5-32.6-7.5-41.2 0-74.6 33.4-74.6 74.5 0 41.2 33.4 74.6 74.6 74.6 38.3 0 69.9-28.9 74.1-66.1h30.2c2.4 0 4.7-1 6.2-2.7l81-82.7c1.1 3 3.3 8.8 6.2 16.1-19 13.5-31.4 35.7-31.4 60.8 0 41.2 33.4 74.6 74.6 74.6 41.2 0 74.6-33.4 74.6-74.6-.1-41.1-33.5-74.5-74.7-74.5zm-240.8 83h57.1c-1.8 12.2-7.4 23.5-16.3 32.4-10.9 10.9-25.4 16.9-40.8 16.9-15.4 0-29.9-6-40.8-16.9-10.9-10.9-16.9-25.4-16.9-40.8 0-15.4 6-29.9 16.9-40.8 10.9-10.9 25.4-16.9 40.8-16.9 15.4 0 29.9 6 40.8 16.9 8.9 8.9 14.5 20.1 16.3 32.4H75.6c-4.7 0-8.4 3.8-8.4 8.4-.1 4.6 3.7 8.4 8.4 8.4zm74.1-16.9c-2.3-20-12.4-37.6-27.2-49.6l16.7-26.3 28.5 75.9h-18zm33.4-6.9l-29.8-79.3h107.2l.1.2-77.5 79.1zm174.1 56.1c-10.9 10.9-25.4 16.9-40.8 16.9-15.4 0-29.9-6-40.8-16.9-10.9-10.9-16.9-25.4-16.9-40.8 0-15.4 6-29.9 16.9-40.8 1.4-1.4 2.9-2.8 4.4-4 0 .1 0 .1.1.2 11.1 25.1 21.4 42.2 30.5 50.8 1.6 1.5 3.7 2.3 5.8 2.3 2.2 0 4.5-.9 6.1-2.6 3.2-3.4 3-8.7-.3-11.9-9-8.5-19.4-28.8-27.5-47.3 6.8-2.8 14.2-4.2 21.7-4.2 15.4 0 29.9 6 40.8 16.9 10.9 10.9 16.9 25.4 16.9 40.8 0 15.3-6 29.7-16.9 40.6z">
            </path>
          </svg>
        </div>
      </td>
    </tr>
    <tr>
      <td>
        BICYCLE_RENT
      </td>
      <td>
        <div>
        </div>
      </td>
    </tr>
    <tr>
      <td>
        BUS
      </td>
      <td>
        <div>
          <svg viewbox="0 0 390 390">
            <path d="m 325.28866,67.290913 c 0,-31.901057 -58.52714,-48.981938 -130.6185,-48.981938 -72.09137,0 -130.492908,17.080881 -130.618502,48.981938 L 56.515975,211.72483 v 90.4282 c 0,5.52615 4.521409,10.04757 10.047577,10.04757 h 8.79163 v 25.11894 c 0,8.28925 6.782114,15.07137 15.071365,15.07137 h 10.047583 c 8.28925,0 15.07136,-6.78212 15.07136,-15.07137 V 312.2006 h 158.24934 v 25.11894 c 0,8.28925 6.78211,15.07137 15.07136,15.07137 h 10.04759 c 8.28925,0 15.07136,-6.78212 15.07136,-15.07137 V 312.2006 h 8.79162 c 5.52617,0 10.04758,-4.52142 10.04758,-10.04757 v -90.4282 z M 148.82809,38.404129 h 91.68414 c 4.89819,0 8.79163,3.893436 8.79163,8.79163 0,4.898194 -3.89344,8.79163 -8.79163,8.79163 h -91.68414 c -4.89819,0 -8.79163,-3.893436 -8.79163,-8.79163 0,-4.898194 3.89344,-8.79163 8.79163,-8.79163 z M 95.450336,269.49839 c -10.424362,0 -18.839207,-8.41483 -18.839207,-18.8392 0,-10.42437 8.414845,-18.83921 18.839207,-18.83921 10.424364,0 18.839204,8.41484 18.839204,18.83921 0,10.42437 -8.41484,18.8392 -18.839204,18.8392 z m 198.439644,0 c -10.42436,0 -18.8392,-8.41483 -18.8392,-18.8392 0,-10.42437 8.41484,-18.83921 18.8392,-18.83921 10.42437,0 18.83921,8.41484 18.83921,18.83921 0,10.42437 -8.41484,18.8392 -18.83921,18.8392 z m 8.16366,-57.77356 H 87.286679 c -6.530924,0 -11.680308,-5.40057 -11.303524,-11.9315 L 82.137296,83.115846 C 82.388485,77.715274 86.7843,73.570648 92.184873,73.570648 H 296.02509 c 6.02855,0 10.92674,4.647004 11.30352,10.675551 l 6.02855,115.547131 c 0.37679,6.53093 -4.77259,11.9315 -11.30352,11.9315 z">
            </path>
          </svg>
        </div>
      </td>
    </tr>
    <tr>
      <td>
        CAR
      </td>
      <td>
        <div>
          <svg viewbox="0 0 390 390">
            <path d="M371.2 175.3l-3.7-12.1-29.8-95.7c-5.8-18.6-22.9-31.2-42.4-31.2H94.7c-19.5 0-36.5 12.6-42.4 31.2l-29.8 95.7-3.7 12.1C7.6 181.4 0 193.1 0 206.5v76.2c0 7.8 6.4 14.2 14.2 14.2h12.4v35.5c0 11.7 9.6 21.3 21.3 21.3H62c11.7 0 21.3-9.6 21.3-21.3v-35.5h223.4v35.5c0 11.7 9.6 21.3 21.3 21.3h14.2c11.7 0 21.3-9.6 21.3-21.3v-35.5h12.4c7.8 0 14.2-6.4 14.2-14.2v-76.2c-.1-13.3-7.7-25.1-18.9-31.2zM55 252.8c-14.7 0-26.6-11.9-26.6-26.6s11.9-26.6 26.6-26.6 26.6 11.9 26.6 26.6-11.9 26.6-26.6 26.6zm240.7-81.6c-9-14.9-25.4-24.8-44-24.8s-34.9 9.9-44 24.8H47.9l29.8-95.6c2.3-7.4 9.2-12.4 16.8-12.4h200.9c7.8 0 14.5 5 16.8 12.4l29.8 95.7-46.3-.1zm39.3 81.6c-14.7 0-26.6-11.9-26.6-26.6s11.9-26.6 26.6-26.6c14.7 0 26.6 11.9 26.6 26.6s-11.8 26.6-26.6 26.6z">
            </path>
            <circle cx="251.7"
                    cy="107.4"
                    r="33.7"
            >
            </circle>
          </svg>
        </div>
      </td>
    </tr>
    <tr>
      <td>
        CAR_PARK
      </td>
      <td>
        <div>
          <svg viewbox="0 0 390 390">
            <path d="M371.2 175.3l-3.7-12.1-29.8-95.7c-5.8-18.6-22.9-31.2-42.4-31.2H94.7c-19.5 0-36.5 12.6-42.4 31.2l-29.8 95.7-3.7 12.1C7.6 181.4 0 193.1 0 206.5v76.2c0 7.8 6.4 14.2 14.2 14.2h12.4v35.5c0 11.7 9.6 21.3 21.3 21.3H62c11.7 0 21.3-9.6 21.3-21.3v-35.5h223.4v35.5c0 11.7 9.6 21.3 21.3 21.3h14.2c11.7 0 21.3-9.6 21.3-21.3v-35.5h12.4c7.8 0 14.2-6.4 14.2-14.2v-76.2c-.1-13.3-7.7-25.1-18.9-31.2zM55 252.8c-14.7 0-26.6-11.9-26.6-26.6s11.9-26.6 26.6-26.6 26.6 11.9 26.6 26.6-11.9 26.6-26.6 26.6zm240.7-81.6c-9-14.9-25.4-24.8-44-24.8s-34.9 9.9-44 24.8H47.9l29.8-95.6c2.3-7.4 9.2-12.4 16.8-12.4h200.9c7.8 0 14.5 5 16.8 12.4l29.8 95.7-46.3-.1zm39.3 81.6c-14.7 0-26.6-11.9-26.6-26.6s11.9-26.6 26.6-26.6c14.7 0 26.6 11.9 26.6 26.6s-11.8 26.6-26.6 26.6z">
            </path>
            <circle cx="251.7"
                    cy="107.4"
                    r="33.7"
            >
            </circle>
          </svg>
        </div>
      </td>
    </tr>
    <tr>
      <td>
        FERRY
      </td>
      <td>
        <div>
          <svg viewbox="0 0 100 68.932">
            <path d="M98.634 68.932a14.44 14.44 0 01-6.058-1.335 14.915 14.915 0 00-6.263-1.402c-2.226 0-4.37.521-6.25 1.402a14.59 14.59 0 01-6.082 1.335c-2.143 0-4.213-.486-6.081-1.335a14.803 14.803 0 00-6.239-1.402c-2.238 0-4.357.521-6.273 1.402a14.475 14.475 0 01-6.059 1.335c-2.155 0-4.238-.486-6.07-1.335a14.918 14.918 0 00-6.262-1.402c-2.227 0-4.371.521-6.25 1.402a14.537 14.537 0 01-6.071 1.335c-2.167 0-4.226-.486-6.095-1.335a14.753 14.753 0 00-6.237-1.402c-2.229 0-4.359.521-6.263 1.402a14.532 14.532 0 01-6.07 1.335v-9.2c2.156 0 4.228-.496 6.07-1.333a14.924 14.924 0 016.263-1.402c2.214 0 4.358.521 6.237 1.402 1.869.837 3.927 1.333 6.095 1.333 2.155 0 4.228-.496 6.071-1.333a14.785 14.785 0 016.25-1.402c2.226 0 4.357.521 6.262 1.402a14.68 14.68 0 006.07 1.333 14.66 14.66 0 006.059-1.333c1.916-.881 4.035-1.402 6.273-1.402 2.214 0 4.348.521 6.239 1.402 1.868.837 3.938 1.333 6.081 1.333 2.166 0 4.227-.496 6.082-1.333a14.792 14.792 0 016.25-1.402c2.227 0 4.358.521 6.263 1.402a14.62 14.62 0 006.058 1.333v9.2zM.048 29.398L.013 51.125c2.156 0 4.228-.483 6.07-1.318a14.865 14.865 0 016.263-1.405c2.214 0 4.358.521 6.237 1.405a14.903 14.903 0 006.095 1.318c2.155 0 4.228-.483 6.071-1.318a14.727 14.727 0 016.25-1.405c2.226 0 4.357.521 6.262 1.405a14.702 14.702 0 006.07 1.318c2.145 0 4.228-.483 6.059-1.318 1.916-.885 4.035-1.405 6.273-1.405 2.214 0 4.348.521 6.239 1.405a14.923 14.923 0 006.081 1.318c.612 0 1.197-.133 1.795-.193 1.08-.242 2.18-.352 3.233-.774l.072-.037c7.83-3.207 14.596-11.171 18.811-17.103l2.106-3.62H.048zM78.22 4.948C75.814-.45 69.336.01 69.336.01H.013L0 25.803h86.913L78.22 4.948zM33.274 20.211c0 .993-.814 1.817-1.808 1.817h-8.009c-.994 0-1.796-.824-1.796-1.817V7.864a1.8 1.8 0 011.796-1.802h8.009c.994 0 1.808.812 1.808 1.802v12.347zm16.655 0c0 .993-.802 1.817-1.785 1.817h-8.071c-.993 0-1.795-.824-1.795-1.817V7.864a1.8 1.8 0 011.795-1.802h8.071a1.8 1.8 0 011.785 1.802v12.347zm-33.25 0c0 .993-.813 1.817-1.796 1.817H6.814c-.994 0-1.795-.824-1.795-1.817V7.864a1.8 1.8 0 011.795-1.802h8.069c.982 0 1.796.812 1.796 1.802v12.347zM55.842 6.062h14.452l.12-.011h.097c.153-.012 3.961-.159 5.244 2.711l5.518 13.267H62.465L55.842 6.062z">
            </path>
          </svg>
        </div>
      </td>
    </tr>
    <tr>
      <td>
        FLEX
      </td>
      <td>
        <div>
        </div>
      </td>
    </tr>
    <tr>
      <td>
        GONDOLA
      </td>
      <td>
        <div>
          <svg viewbox="0 0 390 390">
            <path d="M277.9 206.4c-15.8-5.3-54.8-8.6-74.4-10v-87.9L318.2 150v-15.3l-71.6-25.9V96.9L143.1 59.7v11.9L71.7 46.5v15.2L182 100.8v94.7l-43-.7s-34.9 3.6-55.5 27.8S62 261.1 62 275.4s11.6 38.5 29.6 48.4c17.9 9.8 40.3 11.6 40.3 11.6l95.8 8.1s45.7-1.3 66.3-13.4c15.2-9 34-23.3 34-58.2 0-35-23.8-56.7-50.1-65.5zm-134.4 84.2H84.4s-7.2-26 2.7-42.1 18.8-22.4 18.8-22.4l37.6.9v63.6zm78 0h-57.3V227h57.3v63.6zm86.8 0h-66.1V227h44.6s9 5.4 18.8 21.5c9.9 16.1 2.7 42.1 2.7 42.1z">
            </path>
          </svg>
        </div>
      </td>
    </tr>
    <tr>
      <td>
        MICROMOBILITY
      </td>
      <td>
        <div>
          <svg viewbox="0 0 512 512">
            <path d="M464.027 393.21L416.602 51.747C412.504 22.246 386.953 0 357.172 0H289v40h68.172c9.93 0 18.441 7.414 19.808 17.246l36.149 260.25C361 332.586 321.293 377.22 313.426 432H116.559c-8.254-23.277-30.485-40-56.559-40-33.086 0-60 26.914-60 60s26.914 60 60 60c26.074 0 48.305-16.723 56.559-40H352v-20c0-43.473 27.887-80.55 66.71-94.297l5.7 41.043C405.176 408.75 392 428.86 392 452c0 33.086 26.914 60 60 60s60-26.914 60-60c0-28.965-20.633-53.203-47.973-58.79zM60 472c-11.027 0-20-8.973-20-20s8.973-20 20-20 20 8.973 20 20-8.973 20-20 20zm392 0c-11.027 0-20-8.973-20-20s8.973-20 20-20 20 8.973 20 20-8.973 20-20 20zm0 0">
            </path>
          </svg>
        </div>
      </td>
    </tr>
    <tr>
      <td>
        MICROMOBILITY_RENT
      </td>
      <td>
        <div>
          <svg viewbox="0 0 512 512">
            <path d="M464.027 393.21L416.602 51.747C412.504 22.246 386.953 0 357.172 0H289v40h68.172c9.93 0 18.441 7.414 19.808 17.246l36.149 260.25C361 332.586 321.293 377.22 313.426 432H116.559c-8.254-23.277-30.485-40-56.559-40-33.086 0-60 26.914-60 60s26.914 60 60 60c26.074 0 48.305-16.723 56.559-40H352v-20c0-43.473 27.887-80.55 66.71-94.297l5.7 41.043C405.176 408.75 392 428.86 392 452c0 33.086 26.914 60 60 60s60-26.914 60-60c0-28.965-20.633-53.203-47.973-58.79zM60 472c-11.027 0-20-8.973-20-20s8.973-20 20-20 20 8.973 20 20-8.973 20-20 20zm392 0c-11.027 0-20-8.973-20-20s8.973-20 20-20 20 8.973 20 20-8.973 20-20 20zm0 0">
            </path>
          </svg>
        </div>
      </td>
    </tr>
    <tr>
      <td>
        SCOOTER
      </td>
      <td>
        <div>
          <svg viewbox="0 0 512 512">
            <path d="M464.027 393.21L416.602 51.747C412.504 22.246 386.953 0 357.172 0H289v40h68.172c9.93 0 18.441 7.414 19.808 17.246l36.149 260.25C361 332.586 321.293 377.22 313.426 432H116.559c-8.254-23.277-30.485-40-56.559-40-33.086 0-60 26.914-60 60s26.914 60 60 60c26.074 0 48.305-16.723 56.559-40H352v-20c0-43.473 27.887-80.55 66.71-94.297l5.7 41.043C405.176 408.75 392 428.86 392 452c0 33.086 26.914 60 60 60s60-26.914 60-60c0-28.965-20.633-53.203-47.973-58.79zM60 472c-11.027 0-20-8.973-20-20s8.973-20 20-20 20 8.973 20 20-8.973 20-20 20zm392 0c-11.027 0-20-8.973-20-20s8.973-20 20-20 20 8.973 20 20-8.973 20-20 20zm0 0">
            </path>
          </svg>
        </div>
      </td>
    </tr>
    <tr>
      <td>
        RAIL
      </td>
      <td>
        <div>
          <svg viewbox="0 0 390 390">
            <path d="M257.4 300l39.2-28c5.3-3.8 8.4-9.8 8.4-16.3V129c0-8.6-5.5-16.3-13.7-19-1.1-.4-2.3-.8-3.5-1.1l-5.7-38.3c-1.4-9.1-8-16.6-17-18.9-14.9-3.8-39.8-8.7-70.1-8.7s-55.3 4.9-70.1 8.7c-9 2.3-15.6 9.8-17 18.9l-5.7 38.3c-1.2.4-2.4.8-3.5 1.1-8.2 2.7-13.7 10.4-13.7 19v126.7c0 6.5 3.1 12.5 8.4 16.3l39.2 28L84 372h29l8.1-12h147.8l8.1 12h29l-48.6-72zm31.6-49c0 8.3-6.7 15-15 15s-15-6.7-15-15 6.7-15 15-15 15 6.7 15 15zM202.5 66.5c0-4.6 3.9-8.3 8.5-8 21.5 1.3 39.2 4.9 50.4 7.8 3.1.8 5.5 3.4 5.9 6.6l3.8 25.6c.4 2.8-2.1 5.1-4.9 4.5-15.6-3.8-34.9-7.2-56.1-8.4-4.3-.2-7.6-3.7-7.6-8V66.5zM237.9 314h-85.7l9.4-14h66.8l9.5 14zM214 127c0 10.5-8.5 19-19 19s-19-8.5-19-19 8.5-19 19-19 19 8.5 19 19zm-98 139c-8.3 0-15-6.7-15-15s6.7-15 15-15 15 6.7 15 15-6.7 15-15 15zm7.8-163.1c-2.8.7-5.3-1.7-4.9-4.5l3.8-25.6c.5-3.2 2.8-5.8 5.9-6.6 11.2-2.9 28.8-6.5 50.4-7.7 4.6-.3 8.5 3.4 8.5 8v20c0 4.3-3.3 7.8-7.6 8-21.2 1.1-40.5 4.6-56.1 8.4zm8.1 241.1l9.4-14h107.3l9.4 14H131.9z">
            </path>
            <circle cx="183"
                    cy="29"
                    r="8"
            >
            </circle>
            <circle cx="207"
                    cy="29"
                    r="8"
            >
            </circle>
          </svg>
        </div>
      </td>
    </tr>
    <tr>
      <td>
        STREETCAR
      </td>
      <td>
        <div>
          <svg viewbox="0 0 390 390">
            <path d="M115.4 309.3c3.1 6.6 11.1 12 18.8 13.9L102 371.9h29.4l31.7-48 60.7.1 31.5 48h29.6l-32.3-48.7c7.5-2 15-7.4 18.2-13.9l9.3-18.4c4-8.1 6.9-22.1 6.5-31.2L276.5 98.9c-.6-12-11.2-23.3-25.8-23.3h-23.2c4-10.1 11-27.5 11.7-29.4 10.4.7 12.6 2.1 13 2.4l.6 1.8-.6-1.2c.3 2.7 2.5 4.7 5.1 4.7 2.9 0 6-3.2 6-6 0-8.4-13-11-24.4-12.2-8.6-1-23.7-2.2-45.3-2.2-21.6 0-36.6 1.2-45.3 2.2-11.3 1.2-24.4 3.8-24.4 12.2 0 2.8 3.1 6 5.9 6 2.7 0 4.9-2 5.2-4.7l-.5 1.2s.7-1.7.7-1.8c.3-.4 2.6-1.7 12.9-2.4.7 1.9 7.7 19.3 11.7 29.4h-24.6c-13.8 0-25.9 11.9-25.9 23.3l-9.4 161c-.6 9 2.7 22.8 6.4 31l9.1 18.4zm30.4-5.5c-9.1 0-16.5-7.4-16.5-16.5 0-9 7.4-16.4 16.5-16.4s16.4 7.4 16.4 16.4c0 9.1-7.3 16.5-16.4 16.5zm94.3 0c-9 0-16.3-7.4-16.3-16.5 0-9 7.4-16.4 16.3-16.4 9.1 0 16.5 7.4 16.5 16.4 0 9.1-7.4 16.5-16.5 16.5zM193.6 45.4c6 0 24.3.3 32.9.4-1.5 3.7-8.1 20.1-11.9 29.7h-42c-3.9-9.6-10.4-26.1-11.9-29.7 8.6-.2 27-.4 32.9-.4zM139 109.5c0-7 5.7-12.7 12.6-12.7h82.8c6.9 0 12.6 5.6 12.6 12.7v4.3c0 6.9-5.6 12.6-12.6 12.6h-82.8c-6.9 0-12.6-5.7-12.6-12.6v-4.3zm-20.1 135.3l5.4-94.7c.4-6.9 6.8-12.8 13.7-12.8l111.3.1c6.9 0 12.9 5.7 13.4 12.6l5.6 94.9c.5 6.9-4.9 12.7-11.8 12.7h-126c-6.9-.1-12-5.9-11.6-12.8z">
            </path>
          </svg>
        </div>
      </td>
    </tr>
    <tr>
      <td>
        SUBWAY
      </td>
      <td>
        <div>
          <svg viewbox="0 0 390 390">
            <path d="M257.4 300h7.6c16.6 0 30-13.4 30-30V92c0-16.6-13.4-30-30-30h-56.5l40.3-29.9 11.6 9.9c.8.6 1.7 1 2.6 1 1.1 0 2.2-.5 3-1.4 1.4-1.7 1.2-4.2-.4-5.6l-14-12c-.7-.6-1.6-1-2.6-1H141c-1 0-1.9.3-2.6 1l-14 12c-1.7 1.4-1.9 4-.4 5.6 1.4 1.7 4 1.9 5.6.4l11.6-9.9L181.5 62H125c-16.6 0-30 13.4-30 30v178c0 16.6 13.4 30 30 30h7.6L84 372h26l10.8-16h148.4l10.8 16h26l-48.6-72zm6.6-34c-8.3 0-15-6.7-15-15s6.7-15 15-15 15 6.7 15 15-6.7 15-15 15zM153.1 31h83.8L195 62l-41.9-31zm5.4 45h73c3.9 0 7 3.1 7 7s-3.1 7-7 7h-73c-3.9 0-7-3.1-7-7s3.1-7 7-7zM111 112c0-4.4 3.6-8 8-8h152c4.4 0 8 3.6 8 8v100c0 4.4-3.6 8-8 8H119c-4.4 0-8-3.6-8-8V112zm131.2 204h-94.4l10.8-16h72.8l10.8 16zM126 266c-8.3 0-15-6.7-15-15s6.7-15 15-15 15 6.7 15 15-6.7 15-15 15zm2.9 78l10.8-16h110.6l10.8 16H128.9z">
            </path>
          </svg>
        </div>
      </td>
    </tr>
    <tr>
      <td>
        TRAM
      </td>
      <td>
        <div>
          <svg viewbox="0 0 390 390">
            <path d="M257.4 300h7.6c16.6 0 30-13.4 30-30V92c0-16.6-13.4-30-30-30h-56.5l40.3-29.9 11.6 9.9c.8.6 1.7 1 2.6 1 1.1 0 2.2-.5 3-1.4 1.4-1.7 1.2-4.2-.4-5.6l-14-12c-.7-.6-1.6-1-2.6-1H141c-1 0-1.9.3-2.6 1l-14 12c-1.7 1.4-1.9 4-.4 5.6 1.4 1.7 4 1.9 5.6.4l11.6-9.9L181.5 62H125c-16.6 0-30 13.4-30 30v178c0 16.6 13.4 30 30 30h7.6L84 372h26l10.8-16h148.4l10.8 16h26l-48.6-72zm6.6-34c-8.3 0-15-6.7-15-15s6.7-15 15-15 15 6.7 15 15-6.7 15-15 15zM153.1 31h83.8L195 62l-41.9-31zm5.4 45h73c3.9 0 7 3.1 7 7s-3.1 7-7 7h-73c-3.9 0-7-3.1-7-7s3.1-7 7-7zM111 112c0-4.4 3.6-8 8-8h152c4.4 0 8 3.6 8 8v100c0 4.4-3.6 8-8 8H119c-4.4 0-8-3.6-8-8V112zm131.2 204h-94.4l10.8-16h72.8l10.8 16zM126 266c-8.3 0-15-6.7-15-15s6.7-15 15-15 15 6.7 15 15-6.7 15-15 15zm2.9 78l10.8-16h110.6l10.8 16H128.9z">
            </path>
          </svg>
        </div>
      </td>
    </tr>
    <tr>
      <td>
        TRANSIT
      </td>
      <td>
        <div>
          <svg viewbox="0 0 100.3 100.29"
               height="100%"
               width="100%"
          >
            <path id="path15"
                  d="M100.3,50.15A50.15,50.15,0,1,1,50.15,0,50.15,50.15,0,0,1,100.3,50.15"
                  style="fill: rgb(224, 101, 31);"
            >
            </path>
            <path id="path17"
                  d="M61.62,58.06A22.49,22.49,0,1,1,72.83,38.59"
                  style="fill: none; stroke: rgb(255, 255, 255); stroke-width: 6.08px;"
            >
            </path>
            <path id="path19"
                  d="M50.88,37.61a22.49,22.49,0,1,1-22.32.1"
                  style="fill: none; stroke: rgb(255, 255, 255); stroke-width: 6.08px;"
            >
            </path>
            <path id="path21"
                  d="M38.17,57.17A22.49,22.49,0,1,1,49.41,76.73"
                  style="fill: none; stroke: rgb(255, 255, 255); stroke-width: 6.08px;"
            >
            </path>
          </svg>
        </div>
      </td>
    </tr>
    <tr>
      <td>
        WALK
      </td>
      <td>
        <div>
          <svg viewbox="0 0 390 390">
            <path d="m 207.60225,19.503527 c 16.02,0 29.07,13.05 29.07,29.07 0,16.020001 -13.05,29.07 -29.07,29.07 -16.02,0 -29.07,-13.049999 -29.07,-29.07 0,-16.02 13.05,-29.07 29.07,-29.07 z m 88.65,181.620003 c -4.05,5.94 -12.15,7.47 -18.09,3.42 0,0 -40.23,-27.45 -43.11,-29.43 -2.88,-1.98 -4.59,-6.12 -5.85,-8.82 -0.9,-1.8 -5.4,-10.62 -8.37,-16.47 l -3.6,15.3 -9.36,42.39 c 0,0 50.13,59.04 51.12,60.21 1.44,1.62 2.25,4.95 2.88,7.47 0.63,2.43 13.41,71.91 13.41,71.91 1.98,10.71 -5.13,21.06 -15.84,23.13 -10.71,1.98001 -21.06,-5.13 -23.13,-15.84 l -12.78,-68.67 -41.67,-45.54 c 0,0 -9.27,43.29 -9.45,44.19 -0.18,0.9 -0.63,2.25 -1.17,3.51 -0.54,1.17 -42.84,72.99 -42.84,72.99 -5.58,9.36 -17.73,12.51 -27.18,6.93 -9.359998,-5.58 -12.509998,-17.73 -6.929998,-27.18 l 37.619998,-63.27 30.24,-136.89 -20.34,16.2 -11.25,49.59 c 0,7.02 -7.92,11.61 -14.94,10.08 -7.02,-1.53 -11.16,-4.41 -9.54,-15.39 h 0.18 c 0,0 11.7,-55.08 12.6,-57.51 0.81,-2.43 2.07,-5.13 3.33,-6.12 8.46,-6.84 41.22,-33.66 50.85,-41.400003 11.07,-9.000001 15.93,-12.329999 25.38,-12.329999 7.11,0 14.85,2.159999 23.04,10.62 3.6,3.690001 6.21,9.720002 8.01,13.500002 1.8,3.69 24.39,48.33 24.39,48.33 l 38.97,27 c 5.94,4.05 7.47,12.15 3.42,18.09 z">
            </path>
          </svg>
        </div>
      </td>
    </tr>
    <tr>
      <td>
        NONE_OF_THE_ABOVE
      </td>
      <td>
        <div>
        </div>
      </td>
    </tr>
  </tbody>
</table>
`;
